### DISCUSSION TITLE
Standalone flet app with many dependencies

### DISCUSSION THREAD

**Discussion by noahwoelki** [flet-dev/flet#3627] [Q&A]
### Question

Dear Community,

I have a python project with many functions and specialized implementations. I also have a app implemented in flet which is calling many of these functions from different directories, and also a local database (.sqlite) which is needed by the app.

My question would be: Can i create a standalone app including all the required dependencies and packages that need to be installed, as well as the database? I was looking at https://flet.dev/docs/publish but I could not understand if i can add utility functions or just files like icons etc.

I would be very happy on any examples/resources regarding such a project.

Thanks for any help and best regards!

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Your application can be packaged as standalone as seen in that guide.
Give it a try, and report if you face any issues.
